composer/datasets/in_context_learning_evaluation.py

```diff
@@ -918,7 +919,15 @@ def __init__(
                     'test_outputs': examples['test_outputs'],
                     'language': examples['language'],
                 }))
+
+        if generations_per_sample < pass_at_k:
+            raise ValueError(
+                f'Invalid for generations_per_sample ({generations_per_sample}) to be less than pass_at_k ({pass_at_k}) for code evaluation.'
+            )
```

Comment:
```suggestion
            raise ValueError(
                f'generations_per_sample ({generations_per_sample}) must be greater than or equal to pass_at_k ({pass_at_k}) for code evaluation.'
            )
```
nit: more clearly indicates change required by user

Reply:
nice. works for me.